src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,29 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) INJECTOR
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        //TESTING
+        AuthenticationService authenticationService = (AuthenticationService) INJECTOR
+                                                        .getInstance(AuthenticationService.class);
+        try {
+            User ivan = authenticationService.register("ivan@gmail.com", "qwerty1234");
+            User bob = authenticationService.register("bob@gmail.com", "password123456");
+        } catch (RegistrationException e) {
+            throw new RuntimeException("Can't register the user: ", e);
```

Comment:
instead of throw new RuntimeException("Can't register the user: ", e) it is better to use more specific exception like RegistrationException